ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/evpn_vlan_bundle.cfg

```diff
@@ -0,0 +1,102 @@
+!RANCID-CONTENT-TYPE: arista
+!
+vlan internal order ascending range 1006 1199
+!
+transceiver qsfp default-mode 4x10G
+!
+service routing protocols model multi-agent
+!
+hostname evpn_vlan_bundle
+!
+no enable password
+no aaa root
+!
+vlan 10
+   name storage_prod
+!
+vlan 20
+   name storage_prod
+!
+vlan 50
+   name ipmi
+!
+vlan 60
+   name ipmi
+!
+vrf instance MGMT
+!
+interface Loopback0
+   description EVPN_Overlay_Peering
+   no shutdown
+   ip address 10.10.0.3/32
+!
+interface Loopback1
+   description VTEP_VXLAN_Tunnel_Source
+   no shutdown
+   ip address 10.11.0.3/32
+!
+interface Vxlan1
+   description evpn_vlan_bundle_VTEP
+   vxlan source-interface Loopback1
+   vxlan udp-port 4789
+   vxlan vlan 10 vni 10010
+   vxlan vlan 20 vni 10020
+   vxlan vlan 50 vni 10050
+   vxlan vlan 60 vni 10060
+!
+ip routing
+no ip routing vrf MGMT
+!
+ip prefix-list PL-LOOPBACKS-EVPN-OVERLAY
+   seq 10 permit 10.10.0.0/24 eq 32
+   seq 20 permit 10.11.0.0/24 eq 32
+!
+route-map RM-CONN-2-BGP permit 10
+   match ip address prefix-list PL-LOOPBACKS-EVPN-OVERLAY
+!
+router bfd
+   multihop interval 300 min-rx 300 multiplier 3
+!
+router bgp 65101
+   router-id 10.10.0.3
+   maximum-paths 4 ecmp 4
+   update wait-install
+   no bgp default ipv4-unicast
+   neighbor EVPN-OVERLAY-PEERS peer group
+   neighbor EVPN-OVERLAY-PEERS update-source Loopback0
+   neighbor EVPN-OVERLAY-PEERS bfd
+   neighbor EVPN-OVERLAY-PEERS ebgp-multihop 3
+   neighbor EVPN-OVERLAY-PEERS send-community
+   neighbor EVPN-OVERLAY-PEERS maximum-routes 0
+   neighbor IPv4-UNDERLAY-PEERS peer group
+   neighbor IPv4-UNDERLAY-PEERS send-community
+   neighbor IPv4-UNDERLAY-PEERS maximum-routes 12000
+   redistribute connected route-map RM-CONN-2-BGP
+   !
+   vlan-aware-bundle bundle1
+      rd 10.10.0.3:10050
+      route-target both 10050:10050
+      redistribute learned
+      vlan 50,60
+   !
+   vlan-aware-bundle storage_prod
+      rd 10.10.0.3:10010
+      route-target both 10010:10010
```

Comment:
With this strategy of creating the rd and rt automatically derived from the lowest VLAN, do we not risk service disruption, for example, if I add VLAN 9 to the bundle and cause the rd/rt values to change?

Reply:
I think we could introduce a key called `evpn_vlan_bundle_id`, where the id is leveraged for the rd/rt admin subfield
example with default overlay_rt/rd_type.
evpn_vlan_bundle_id: 500
   rd 10.10.0.3:500
   route-target both 500:500

Reply:
Agreed, it is a caveat we already have today if you use the same name on multiple l2vlans.
It would not make sense to set this id under each l2vlan. Maybe we need to build a completely new data structure to cover this; `l2vlan_aware_bundles` and then defining the vlans and even allowing vlan ranges there (we have a request for that).

Reply:
also thought of introducing an option for specifically setting the rt/rd, but did follow the current way of implementation. Today when the vlan bundle is based on the vlan name it is the same, the rt / rd is based on the lowest vlan id and might change if you introduce a new vlan with the same name and lower id.

Reply:
Nevertheless I think it would make sense to at least provide an option to statically set the rd / rt for the vlan-aware-bundles. Should I implement it in this PR or better create a separate issue and PR?

Reply:
It would lead to conflicts if we allow setting it under the vlan itself, since you could have the same name but different ids of different vlans. So I think we need to build another data model. Let me discuss with Carl and get back.